.github/ISSUE_TEMPLATE/setup-mydevelopment.yaml

```diff
@@ -0,0 +1,176 @@
+        the`emails/` directory which should be used to generate the contents of these emails in HTML for copy and pasted
+        into Gmail and [scheduled](https://support.google.com/mail/answer/9214606?hl=en&co=GENIE.Platform%3DDesktop)
+
+        | Schedule        | File                            | Description                                                                                                               |
+        |:----------------|:--------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+        | 2 week reminder | `attendance_reminder_email.Rmd` | An attendance reminder to encourage participants who cannot attend anymore to cancel.                                     |
+        | 1 Week Reminder | `setup_email.Rmd` / `setup_email_two_day.Rmd` | A reminder of the setup instructions as well as information about the session platform and joining link (use appropriately to how course is being run). |
+        | 4 Day Reminder  | `setup_email.Rmd` / `setup_email_two_day.Rmd` | A reminder of the setup instructions as well as information about the session platform and joining link (use appropriately to how course is being run). |
+        | On the Morning  | `joining_reminder_email.Rmd`    | A reminder of the joining link.                                                                                           |
+        | Day After       | `feedback_survey_email.Rmd`     | A request to complete a survey.                                                                                           |
+
+        Not all templates require all parameters but across all four email templates the parameters required are:
+
+        - `lead_instructor`: name of the lead instructor.
+        - `session_joining_link`: link to the blackboard collaborate room (if teaching on-line/hybrid).
+        - `event_date`: the date of the session in ISO format (`YYYY-MM-DD`)
+        - `event_start_time`: defaults to "10:00"
+        - `lunch_time`: defaults to "13:00"
+        - `prefilled_survey_link`: the unchanging part of the survey link containing the identifier of the date of course question to be pre-populated with the `event_date` parameter.
+
+        **IMPORTANT** If the course is being run _only_ in-person you should remove **ALL** references to joining online or using two monitors from the emails to avoid confusion and the possibility of some people joining remotely.
+
+        #### Knitting & Sending
+
+
+        To use these, open each `.Rmd`, edit parameters in the YAML header under `params` and knit to render to html. If you do this in [RStudio](https://rstrudio.com) then you can select the "_Knit on Save_" option. Alternatively you can knit the document from [R](https://www.r-project.org) with the following, assuming your currently located in the root of the repository directory and wish to knit the `<email-to-be-knitted>.Rmd`.
```

Comment:
```suggestion
        To use these, open each `.Rmd`, edit parameters in the YAML header under `params` and knit to render to html. If you do this in [RStudio](https://rstrudio.com) then you can select the "_Knit on Save_" option. Alternatively you can knit the document from [R](https://www.r-project.org) with the following, assuming you're currently located in the root of the repository directory and wish to knit the `<email-to-be-knitted>.Rmd`.
```

Reply:
You're a spelling/grammar don @tdjames1 thank you :pray: